src/providers/temperature.toit

```diff
@@ -5,14 +5,30 @@
 import system.services
 import ..temperature as api
 
-interface TemperatureProvider:
+interface TemperatureSensor:
```

Comment:
Hmm. There is something iffy about this. I kinda liked that you could have one service provider with multiple handlers. Now, you sort of have to have multiple providers.

Reply:
In the worst case, users have to write their own providers.
But I will try to do more refactorings to make this easy again.